server/user.go

```diff
@@ -43,15 +43,41 @@ func (c *Connection) JiraAccountID() types.ID {
 }
 
 type ConnectionSettings struct {
-	Notifications bool `json:"notifications"`
+	Notifications                bool  `json:"notifications"`
+	SendNotificationsForMention  *bool `json:"send_notifications_for_mention"`
+	SendNotificationsForAssignee *bool `json:"send_notifications_for_assignee"`
+	SendNotificationsForReporter *bool `json:"send_notifications_for_reporter"`
+	SendNotificationsForWatching *bool `json:"send_notifications_for_watching"`
 }
 
 func (s *ConnectionSettings) String() string {
-	notifications := "off"
+	assigneeNotifications := "Notifications for assignee : off"
```

Comment:
I think this looks better without the space before `:`

```suggestion
	assigneeNotifications := "Notifications for assignee: off"
```